### PR TITLE
Rewrite to new RebornCore Power API, texture fixes.

### PR DESCRIPTION
**Pull request by Dragon2488**
Repair commit.

### CODE REVIEW

**Comment by Dragon2488** [comment]
This commit pretended to be in last pull request, however i already pushed it into my fork.

**Comment by Dragon2488** [comment]
Ahh. I'm very bad at git.

**Comment by Dragon2488** [comment]
Don't know how to repair it; will fix manually.

File: src/main/resources/assets/techreborn/blockstates/lightningrod.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
       "model":"orientable",
       "textures":{
          "particle": "techreborn:blocks/machine/generators/lightning_rod_side_off",
-         "top": "techreborn:blocks/machine/generators/lightning_rod_top",
+         "top": "techreborn:blocks/machine/generators/gas_generator_top",
```
Comment (Yulife) [inline src/main/resources/assets/techreborn/blockstates/lightningrod.json:8]: What? Why change it to gas_generator lol

Comment (Dragon2488) [inline src/main/resources/assets/techreborn/blockstates/lightningrod.json:8]: @Yulife I can't find a better texture

Comment (Yulife) [inline src/main/resources/assets/techreborn/blockstates/lightningrod.json:8]: Well I have textures for it anyway

**Pull request closed without merging**